source/_docs/bots-and-indexing.md

```diff
@@ -48,16 +48,24 @@ The `pantheonsite.io` domains are intended for development use and cannot be use
 
 You can index your site under your production domain once it's added to the Live environment. There are many contrib module options available for creating sitemaps for Drupal, including [XMLSiteMap](https://drupal.org/project/xmlsitemap) and [Site_Map](https://drupal.org/project/site_map). WordPress users can install the [Google XML Sitemaps](http://wpcrux.com/collectives/wordpress-xml-sitemap-plugins/) plugin or the [Yoast SEO](https://wordpress.org/plugins/wordpress-seo/), which will maintain sitemap updates automatically. It is up to you to configure the extensions to work as you desire. Pantheon does not offer support for Drupal modules or WordPress plugins.
 
-### Troubleshooting
+## Troubleshooting
 
-#### Sitemaps Produce a White Screen of Death (WSOD)
+### Google Links to My Site over HTTPS with Pantheon's Certificate
+
+Google's search engine will [index HTTPS pages by default](https://webmasters.googleblog.com/2015/12/indexing-https-pages-by-default.html). Because Pantheon serves all environments at `pantheonsite.io` over HTTPS by using our own certificate, this can lead to certificate mismatch security warnings in Google search results when you don't use your own HTTPS certificates. There are several possible solutions, including:
+
+ - adding an `X-Robots-Tag: noindex` header to responses resulting from `https://` requests,

```

Comment:
Meaning that this won't fix existing search indexes from Google, but it can prevent it from happening, and be a slow-to-fix solution long-term, right?